include/deal.II/matrix_free/fe_evaluation.h

```diff
@@ -264,7 +264,19 @@ class FEEvaluationBase
    * by the current cell are overwritten. Thus, if a degree of freedom is
    * associated to more than one cell (as usual in continuous finite
    * elements), the values will be overwritten and only the value written last
-   * is retained. The mask can be used to suppress the write access for some
+   * is retained. Please note that in a parallel context this function might
+   * also touch non-local degrees of freedom, so that a subsequent update or
```

Comment:
```suggestion
   * also touch degrees of freedom owned by other MPI processes, so that a subsequent update or
```

Reply:
What about the other functions, do it here or elsewhere?

Reply:
Let us do it here. I agree with that suggestion.